### PR TITLE
Mid.ProviderHealthInsurance Validation

### PR DESCRIPTION
**Pull request by oajimenez**
Cannot do sample validation because of no matching PKs. Aggregate validation looks good overall: similar total rows and nulls within margin. There are 3 columns with a large margin of distincts but it's hard to trace why this is happening without a sample validation. I checked the stored procedure line by line and with the exception of changing a join to a right join like it was in SQL Server the rest of the code looks good to me.

### CODE REVIEW

File: migration_original/ODS1Stage/tables/Mid/ProviderHealthInsurance/spu_translated_ProviderHealthInsurance.sql [hunk 1]
```diff
@@ -38,55 +38,56 @@ declare
...
-                                hipay.payorname != hipt.productname
-                            group by
-                                hipay.payorcode
-                        )
+select_statement := $$  
+                    with CTE_ProviderBatch as (
+                        select
+                            p.providerid
+                        from
+                            mdm_team.mst.Provider_Profile_Processing as ppp
```
Comment (carresemata) [inline migration_original/ODS1Stage/tables/Mid/ProviderHealthInsurance/spu_translated_ProviderHealthInsurance.sql:46]: please change this to mdm_db

File: migration_original/ODS1Stage/tables/Mid/ProviderHealthInsurance/spu_translated_ProviderHealthInsurance.sql [hunk 2]
```diff
@@ -106,16 +107,16 @@ update_statement := ' update
...
+                            or target.plandisplayname != source.plandisplayname
+                            or target.planname != source.planname
+                            or target.plantypedescription != source.plantypedescription
+                            or target.plantypedisplaydescription != source.plantypedisplaydescription
+                            or target.productname != source.productname
+                            or target.providerid != source.providerid
+                            or target.searchable != source.searchable';
 
 --- insert Statement
 insert_statement := ' insert
```
Comment (carresemata) [inline migration_original/ODS1Stage/tables/Mid/ProviderHealthInsurance/spu_translated_ProviderHealthInsurance.sql:122]: the HealthInsurancePayorID should be created as a uuid_string here in the insert not in the select

**Review by carresemata — CHANGES_REQUESTED** [review]

File: migration_original/ODS1Stage/tables/Mid/ProviderHealthInsurance/spu_translated_ProviderHealthInsurance.sql (hunk 2 above)
Comment (oajimenez) [reply migration_original/ODS1Stage/tables/Mid/ProviderHealthInsurance/spu_translated_ProviderHealthInsurance.sql:122]: This is a Mid table so we do not insert ids here.

**Review by carresemata — APPROVED** [review]
gtg

**Pull request merged**